package.json

```diff
@@ -167,7 +169,7 @@
     "lint:js:flow": "babel-node scripts/flow-check.js",
     "lint:js": "npm run lint:js:eslint && npm run lint:js:flow",
     "lint:css": "stylelint \"+(boilerplate|docs)/web_modules/**/*.css\"",
-    "lint": "npm run lint:js && npm run lint:css",
+    "lint": "npm-run-all --parallel lint:css lint:js:*",
```

Comment:
I tried this on another project

```
    "lint:js:eslint": "eslint --ignore-path .gitignore --fix .",
    "lint:js:flow": "flow src",
    "lint": "npm-run-all --parallel lint:*",
```

Got

```
ERROR: Task not found: "lint:*"
```

With 

```
    "lint:js:eslint": "eslint --ignore-path .gitignore --fix .",
    "lint:js:flow": "flow src",
    "lint:js": "npm-run-all --parallel lint:js:*",
    "lint": "npm-run-all --parallel lint:*",
```

Not duplicate output. So are you sure?